### PR TITLE
User can add to order

### PR DESCRIPTION
**Pull request by magnus-thor**
### https://www.pivotaltracker.com/story/show/152670825 ###
-------------
- User can add to order

### CODE REVIEW

File: src/app/service/shoppingcart.service.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      'product name': p_name,
+      'price': Number(price),
+      'ingredient id': i_id,
+      'ingredient name': i_name
+    });
+    alert(`${p_name} added to cart`);
+    console.log(this.cart);
+  };
+
+// removeCart(searchId: string) {
```
Comment (olofvi) [inline src/app/service/shoppingcart.service.ts]: Are these comments supposed to be in the PR

File: src/app/service/shoppingcart.service.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+@Injectable()
+
+export class ShoppingCartService {
+  cart: any = [];
+
+  // getCart(){
+  //   return Promise.resolve(this.cart);
+  // };
+
```
Comment (olofvi) [inline src/app/service/shoppingcart.service.ts]: remove commented out code?

File: src/app/app.component.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { Component } from '@angular/core';
 import { Product } from './models/product';
 import { ProductService } from './service/product.service';
+import { ShoppingCartService } from './service/shoppingcart.service';
```
Comment (olofvi) [inline src/app/app.component.ts:4]: nice work making a service I think it's gets a cleaner code

**Review by olofvi — COMMENTED** [review]
No tests for the shopping cart service?

File: src/app/service/shoppingcart.service.ts (hunk 1 above)
Comment (magnus-thor) [reply src/app/service/shoppingcart.service.ts]: no, I will take them out

File: src/app/app.component.html [hunk 4]
```diff
@@ -21,80 +21,111 @@
...
 <footer class="footer">
-    <span class="footer-text">
-      <h5> Mån-Fre 11-19, Lör: 12-16.30 </h5>
-      <h5> Beställ mellan 11-18:30 </h5>
-      <h5> Telefon: 076-633 35 52 </h5>
-      <a target="_blank" href="https://goo.gl/maps/s523jHUZ3VL2"><h5> Sveavägen 13, 111 57 </h5></a>
-      <a target="_blank" href="https://www.facebook.com/kimchistan/"><i class="fa fa-facebook-square fa-4x" aria-hidden="true"></i></a>
-    </span>
+  <span class="footer-text">
+  <h5> Mån-Fre 11-19, Lör: 12-16.30 </h5>
```
Comment (olofvi) [inline src/app/app.component.html:124]: Why are we not displaying the opening times on sundays ?

File: src/app/service/shoppingcart.service.spec.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    service = s;
+  }));
+
+  it('addToCart: add items to cart', () => {
+    expect(service.cart.length).toEqual(0);
+
+    service.addToCart(10, 'Bibimbap', 112, 30, 'Chicken');
+
+    expect(service.cart.length).toEqual(1);
+    expect(service.cart[0]).toEqual({product_id: 10, product_name: 'Bibimbap', price: 112, ingredient_id: 30, ingredient_name: 'Chicken'});
```
Comment (olofvi) [inline src/app/service/shoppingcart.service.spec.ts:21]: nice work on the testing :)

**Review by olofvi — COMMENTED** [review]
just a little comments

File: src/app/app.component.html (hunk 4 above)
Comment (nomad-alt) [reply src/app/app.component.html:124]: It's closed on Sundays :)

Comment (olofvi) [reply src/app/app.component.html:124]: then they need to fix they're facebook info ;)

**Pull request merged**